src/bodywork/cli/cli.py

```diff
@@ -523,7 +532,7 @@ def secret(args: Namespace) -> None:
         display_secrets(
             BODYWORK_DEPLOYMENT_JOBS_NAMESPACE,
             group,
-            secret=name if name != "" else None,
+            name,
```

Comment:
Would this prevent us from seeing all secrets in a group?

This would make it tricky to keep track of what secrets have been created.

Reply:
I reckon `name if name else None` ought to do it as the default is `None` as we've established elsewhere.

Reply:
I'm lost here, what you're suggesting will have no effect. `name` gets passed through whether it's `none` or not at the moment. Also currently it allows all secrets in a group to be listed by doing `display secret --group Prod`.

Am I missing something here?

Reply:
It's me that's the problem here. I need to stop reviewing code on trains - need a big screen to do it properly.

Reply:
It's good - sorry!

Reply:
No worries! I'd struggle to do it on the train as well but for me it's more because there's too many distractions.